src/streaming/models/CmcdModel.js

```diff
@@ -440,6 +497,14 @@ function CmcdModel() {
         }
     }
 
+    function _probeNextRequest(mediaType) {
+        if (!streamProcessors || streamProcessors.length === 0) return;
+        for (let streamProcessor of streamProcessors) {
+            if (streamProcessor.getType() !== mediaType) continue;
```

Comment:
Suggest to rather use .filter or while here instead of "continue"

Reply:
Whats the advantage of filter or while over the for loop?